docs/current/cloud/572923-get-started.md

```diff
@@ -0,0 +1,313 @@
+---
+slug: /cloud/572923/get-started
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+# Get Started with Dagger Cloud
+
+## Introduction
+
+Dagger Cloud provides pipeline visualization, operational insights, and distributed caching for your Daggerized pipelines.
+
+This guide helps you get started with Dagger Cloud. Here are the steps you'll follow:
+
+- Sign up for Dagger Cloud
+- Connect Dagger Cloud with your CI provider or CI tool
+- Visualize CI runs with Dagger Cloud
+- Improve CI performance with Dagger Cloud's distributed caching
+
+The next sections will walk you through these steps in detail.
+
+## Prerequisites
+
+This guide assumes that:
+
+- You have an understanding of how Dagger works. If not, [read the Dagger Quickstart](../quickstart/index.mdx).
+- You have a GitHub account (required for Dagger Cloud identity verification). If not, [register for a free GitHub account](https://github.com/signup).
+- You have a source code repository and a Dagger pipeline that interacts with it. If not, follow the steps in Appendix A to [create and populate a GitHub repository with a sample application and Dagger pipeline](#appendix-a-create-a-dagger-pipeline).
+
+## Step 1: Sign up for Dagger Cloud
+
+:::info
+At the end of this step, you will have signed up for Dagger Cloud and obtained a Dagger Cloud token. If you already have a Dagger Cloud account and token, you may skip this step.
+:::
+
+Follow the steps below to sign up for Dagger Cloud, create an organization and obtain a Dagger Cloud token.
+
+1. Browse to the [Dagger Cloud website](https://www.dagger.io/cloud/signup). Click *Continue with GitHub* to log in with your GitHub account.
+
+  ![Authenticate with GitHub](/img/current/cloud/get-started/connect-github.png)
+
+1. On the GitHub authorization screen, confirm the Dagger Cloud connection for authentication. Once authorized, you will be redirected to a welcome page and prompted to create a new Dagger Cloud organization. Enter a name for your organization in the *Organization Name* field. Click *Next* to proceed.
+
+  :::info
+  The organization name can only contain alphanumeric characters and dashes and is unique across Dagger Cloud.
+  :::
+
+  ![Create Dagger Cloud organization](/img/current/cloud/get-started/create-organization.png)
```

Comment:
When reviewing I noticed that this image and a few others looking blurry. The original images are not blurry. Are the images being artificially upsized to fit the page @crjm?

Reply:
@slumbering - please see DEV-2652

Reply:
> When reviewing I noticed that this image and a few others looking blurry. The original images are not blurry. Are the images being artificially upsized to fit the page @crjm?

Correct, images are upsized

Reply:
I've pushed the fix. Like I said to @vikram-dagger I've also decrease the size of other PNG files without compromising their quality.